en/concept/flight_modes.md

```diff
@@ -1,111 +1,130 @@
+Not all flight modes are available (or makes sense), on all vehicle types, and some modes behave differently on different vehicle types.
 
-The sections below provide an overview of the modes, followed by a [flight mode evaluation diagram](#flight-mode-evaluation-diagram) that shows the conditions under which PX4 will transition into a new mode.
+## PX4 Flight Modes
 
-::: info
 User-facing flight mode documentation can be found in:
 
 - [Flight Modes (Multicopter)](../flight_modes_mc/index.md)
 - [Flight Modes (Fixed-Wing)](../flight_modes_fw/index.md)
 - [Flight Modes (VTOL)](../flight_modes_vtol/index.md)
 - [Drive Modes (Differential Rover)](../flight_modes_rover/differential.md)
 - [Drive Modes (Ackermann Rover)](../flight_modes_rover/ackermann.md)
+- [Basic Configuration > Flight Modes](../config/flight_mode.md)
+
+## ROS 2 Flight Modes
+
+Point to the ROS 2 docs.
+
+## Choosing between PX4 and ROS 2 flight Modes
+
+<!-- Format  -->
+
+- if low-level access and or strict timing or high update rate requirements (e.g. direct motor controls on an MC) -> PX4
+- if vehicle has no companion -> PX4
+- if not want to or can use ROS for whatever reason -> PX4
+- if safety-critical (like RTL) -> PX4 (but you might still want to have a fancier RTL replacement on the ROS side)
+
+for the rest: ROS. Specifically, reasons for ROS:
+
+- easier to implement (no need to deal with low-level embedded constraints and requirements, like stack sizes).
+- Easier to maintain (no need to rebase custom PX4 changes).
+- Crashing the app does not crash the vehicle.
+- High-level functionality is available (e.g. dynamic data structures, or many (ROS) libraries).
+- More available compute to do more advanced processing (e.g. computer vision).
+
+## Flight mode restrictions
+
+Some flight modes make sense only under specific pre-flight and in-flight conditions (e.g. GPS lock).
+The system will not allow transitions to those modes until the right conditions are met.
+
+- Flight mode restrictions in PX4: https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/commander/ModeUtil/mode_requirements.cpp#L46
+- Flight mode restrictions in ROS 2 - point to ROS 2 docs.
 
-:::
 
-## Flight Mode Summary
+## PX4 Flight Modes
 
-### Manual Flight Modes
+Something here about what they are, how they related to [Flight Tasks](../concept/flight_tasks.md), where they are located.
+How they are added.
 
-"Manual" modes are those where the user has direct control over the vehicle via the RC control (or joystick).
-Vehicle movement always follows stick movement, but the level/type of response changes depending on the mode.
-For example, experienced fliers can use modes that provide direct passthrough of stick positions to actuators, while beginners will often choose modes that are less responsive to sudden stick-position changes.
+## MAVLink Integration
 
-- **Rovers / Boats:**
+### Standard Modes Protocol
```

Comment:
@bkueng This section is underway.

Some questions you may choose to answer, though it is early for a review.

- I can see that when you add a new mode, you don't need anything special for the current mode to update. 
- Do you need to do anything if you add a new mode to trigger the mode monitor?
- I recall for ROS2 registration will add a mode to the list of available modes. Do you need to do anything in P4 to add a new mode to the list of available modes?